### PR TITLE
Fix a numerous spelling or styling mistakes

### PR DESCRIPTION
**Pull request by paniagua**
this is @stenver's PR.

### CODE REVIEW

File: _includes/visitor_api.md [hunk 1]
```diff
@@ -260,7 +266,7 @@ The explanation of parameters is as follows:
     ?name=Visitor+Name # This name will be saved as the visitor name
     ?email=visitor@email.com # This email will be saved as the visitor email
     ?phone=56565677 # This phone will be saved as the visitors phone
-    ?notes=Some+notes+about+the+visitor # This message will be saved as the visitor notes
+    ?note=Some+note+about+the+visitor # This message will be saved as the visitor note
```
Comment (paniagua) [inline _includes/visitor_api.md]: as the visitor note => as the visitor's note. Prob same for the previous fields.

File: _site/index.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <head>
   <meta charset="utf-8" />
   <meta http-equiv="X-UA-Compatible" content="chrome=1" />
-  <title>Developed by SaleMove</title>
+  <title>Developer by SaleMove</title>
```
Comment (paniagua) [inline _site/index.html]: this can change to ```Powered by``

File: _site/index.html [hunk 3]
```diff
@@ -18,20 +18,20 @@
 
 
 
-<h1 id="salemove-public-api">SaleMove public API.</h1>
+<h1 id="salemove-public-api">Salemove public API.</h1>
```
Comment (paniagua) [inline _site/index.html]: why did it change to lower case ```M```

File: _site/visitor.html [hunk 4]
```diff
@@ -319,7 +325,7 @@ <h2 id="show-notification-and-save-visitor-contact-information-when-he-first-com
 ?name=Visitor+Name # This name will be saved as the visitor name
 ?email=visitor@email.com # This email will be saved as the visitor email
 ?phone=56565677 # This phone will be saved as the visitors phone
-?notes=Some+notes+about+the+visitor # This message will be saved as the visitor notes
+?note=Some+note+about+the+visitor # This message will be saved as the visitor note
```
Comment (paniagua) [inline _site/visitor.html]: visitor note -> visitor's note

**Comment by stenver** [comment]
Updated. I will merge right away to move on to the next update.

**Pull request merged**